api/src/main/resources/workbench.yaml

```diff
@@ -184,6 +276,43 @@ definitions:
     description: levels of access to data in the curated data repository
     enum: &DATA_ACCESS_LEVEL [registered, protected]
 
+  WorkspaceListResponse:
+    type: "object"
+    required:
+     - "items"
+    properties:
+      items:
+        type: "array"
+        items:
+           $ref: "#/definitions/Workspace"
+
+  Workspace:
+    type: "object"
+    required:
+     - "name"
+     - "cohorts"
+    properties:
+      id:
+        type: string
+      name:
+        type: string
+      namespace:
+        type: string
+      description:
+        type: string
+      creator:
+        type: string
+      creationTime:
+        type: string
+        format: date-time
+      lastModifiedTime:
+        type: string
+        format: date-time
```

Comment:
If we were following this:

https://docs.google.com/presentation/d/1War5JFocSoa6pP9r5dmTzRdizsXAf8VCRzypR9hVZ28/edit#slide=id.g1a70b8af01_0_144

it would be an enum primarily, with other fields like "specificDisease", "population", and maybe "purposeDescription".

I don't know if we've settled on that UX; if you want to punt on this for now that's fine.